x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc

```diff
@@ -83,8 +83,8 @@ To set up the watch:
     "search": {
       "request": {
         "indices": [
-          "<logstash-{now-1h}>", <1>
-          "<logstash-{now}>"
```

Comment:
With ILM there will be an alias by default named `logstash`. Rather then trying to query the concrete indices, I think the example should just use a single index `logstash` and the note below call out that `logstash` is an alias. 

In general we should avoid examples that include both time based (e.g. {now}) and rollover based (e.g. -00001) in favor of the fronting alias. The time stamp on the index when combined with rollover changes meaning such that it is the time when the index was created, not necessarily reflective of the content of that index (the content is dependent on the rollover rules). It is a distinction difficult to describe and the alias should be the preferred access across multiple rolled over indices.

Reply:
Thanks @jakelandis. I've updated this and the following snippet to use the `logstash` index alias with 2c1d2db.

Also added some more context around Logstash ILM defaults in the callout text.